query-graphs/lib/json.js

```diff
@@ -39,13 +39,18 @@ function convertChildren(node) {
 }
 
 // Load a JSON tree
-export function loadJson(graphString, _graphCollapse) {
+export function loadJson(json) {
+    var root = {name: "root", children: convertChildren(json)};
+    return {root: root};
+}
+
+// Load a JSON tree from text
+export function loadJsonFromText(graphString, _graphCollapse) {
```

Comment:
Refactoring seems fine. Does it have any functional impact?

Reply:
Same as above